core/src/main/java/com/datastax/oss/driver/internal/core/tracker/MultiplexingRequestTracker.java

```diff
@@ -94,9 +98,12 @@ public void onError(
       long latencyNanos,
       @NonNull DriverExecutionProfile executionProfile,
       @Nullable Node node,
-      @NonNull String logPrefix) {
+      @NonNull String logPrefix,
+      @NonNull ExecutionInfo executionInfo) {
```

Comment:
Here `executionInfo` is marked `@NonNull` but `RequestTracker#onError` does not have that annotation (arguably it should be `@Nullable` since it looks like `CqlRequestHandler` may pass it as `null`)

Reply:
Yes, should be @Nullable, fixed.